repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
Thanks. Actually I thought about it and configuring it would not be hard using options.json. We'd just ask the users to point to a URL with their config file, which is the common way things are handled in Universe. This would be a fairly quick fix, right, or am I missing something?